Products/ZenModel/ZenossInfo.py

```diff
@@ -279,6 +281,7 @@ def getZopeVersion(self):
         major, minor, micro, status, release = version.getZopeVersion()
         return Version(name, major, minor, micro)
 
+    @FunctionCache('getZenossRevision')
```

Comment:
Removing the method is fine, you just have to find all uses of the method.

I'm not sure I understand the concern with pkg_resources